solr/4.7.2.wso2v1/pom.xml

```diff
@@ -0,0 +1,328 @@
+		</dependency>
+		<dependency>
+			<groupId>org.apache.lucene</groupId>
+			<artifactId>lucene-analyzers-phonetic</artifactId>
+			<version>${lucene.version}</version>
+		</dependency>
+		
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>${version.commons.io}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpmime</artifactId>
+			<version>${version.httpmime}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.noggit</groupId>
+			<artifactId>noggit</artifactId>
+			<version>${version.noggit}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.zookeeper</groupId>
+			<artifactId>zookeeper</artifactId>
+			<version>${version.zookeeper}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.spatial4j</groupId>
+			<artifactId>spatial4j</artifactId>
+			<version>${version.spatial4j}</version>
+		</dependency>
+	</dependencies>
+
+	<distributionManagement>
+		<repository>
+			<id>wso2.releases</id>
+			<name>WSO2 internal Repository</name>
+			<url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+		</repository>
+		<snapshotRepository>
+			<id>wso2.snapshots</id>
+			<name>Apache Snapshot Repository</name>
+			<url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>2.3.2</version>
+				<configuration>
+					<source>1.5</source>
+					<target>1.5</target>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<version>2.3.7</version>
+				<configuration>
+					<instructions>
+						<Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+						<Embed-Dependency>
+							lucene-core;scope=compile|runtime,
+							lucene-codecs;scope=compile|runtime,
+							lucene-analyzers-common;scope=compile|runtime,
+							lucene-queries;scope=compile|runtime,
+							lucene-queryparser;scope=compile|runtime,
+ 							lucene-highlighter;scope=compile|runtime,
+							lucene-memory;scope=compile|runtime,
+							lucene-misc;scope=compile|runtime,
+							lucene-expressions;scope=compile|runtime,
+							lucene-grouping;scope=compile|runtime,
+							lucene-join;scope=compile|runtime,
+							lucene-spatial;scope=compile|runtime,
+							lucene-suggest;scope=compile|runtime,
+							lucene-analyzers-phonetic;scope=compile|runtime,
+							commons-io;scope=compile|runtime,
+							httpmime;scope=compile|runtime,
+							zookeeper;scope=compile|runtime,
+							noggit;scope=compile|runtime,
+							spatial4j;scope=compile|runtime
+						</Embed-Dependency>
+						<Export-Package>
+							org.apache.solr.*;version="${solr.version}",
+							org.tartarus.*
```

Comment:
In what ways does the org.tartarus package is relevant to solr for it to be include as part of solr orbit?